conans/cli/command.py

```diff
@@ -104,6 +101,31 @@ def doc(self):
     def parser(self):
         return self._parser
 
+    def _format(self, parser, info, *args):
+        parser_args = parser.parse_args(*args)
+        try:
+            formatarg = parser_args.format
+        except AttributeError:
+            return
+
+        if formatarg is None:
+            return
+
+        filename = formatarg if "." in formatarg else None
+        formatext = formatarg.rsplit(".")[-1]
```

Comment:
```suggestion
        format_ext = formatarg.rsplit(".")[-1]
```

otherwise it is read "forma-text"

Reply:
Removed file handling